addons/web/static/src/webclient/actions/action_service.js

```diff
@@ -1502,6 +1510,9 @@ export function makeActionManager(env, router = _router) {
             }
             const { actionRequest, options } = actionParams;
             options.index = index;
+            if (controller.nextAction) {
+                return doAction(controller.nextAction, options);
+            }
```

Comment:
why do you need this? Isn't setting `controller.action = nextAction` enough?

Reply:
When we restore, we don't use the information on the controller.action, but we use the controller.state, and the controller state wasn't modified and it has the information as it was on the url, in this case it will be action="crm". And nothing more.

I didn't find a way to correctly identify if we have a "real" action in the controller, or just a "virtual" one, so I added the next one, to be sure.